src/rgw/rgw_rest_s3.cc

```diff
@@ -536,10 +536,10 @@ static void dump_usage_categories_info(Formatter *formatter, const rgw_usage_log
     const rgw_usage_data& usage = uiter->second;
     formatter->open_object_section("Entry");
     formatter->dump_string("Category", uiter->first);
-    formatter->dump_int("BytesSent", usage.bytes_sent);
-    formatter->dump_int("BytesReceived", usage.bytes_received);
-    formatter->dump_int("Ops", usage.ops);
-    formatter->dump_int("SuccessfulOps", usage.successful_ops);
+    formatter->dump_unsigned("BytesSent", usage.bytes_sent);
+    formatter->dump_unsigned("BytesReceived", usage.bytes_received);
+    formatter->dump_unsigned("Ops", usage.ops);
+    formatter->dump_unsigned("SuccessfulOps", usage.successful_ops);
```

Comment:
it's probably better to use `encode_json()` in general, ie:
```c++
encode_json("SuccessfulOps", usage.successful_ops, formatter);
```
which has overloads that call `dump_int()` for signed types and `dump_unsigned()` for unsigned types

Reply:
ping @joke-lee